templates/experiments/components/experiment_chat.html

```diff
@@ -135,7 +134,7 @@ <h3 class="font-bold text-lg">Translate Remaining Messages?</h3>
                         hx-vals='{"language": "{{ language }}"}'
                         hx-include="[name='search'], [name='language'], [name='show_original_translation'], #translation-provider-remaining, #translation-provider-model-remaining"
                         hx-target="#messages-container"
-                        hx-indicator="#chat-message-untranslated"
+                        hx-indicator=".translation-loading-spinner"
```

Comment:
I don't see anything with this exact class. I do see `translation-loading-spinner-{{ message.id }}` but that won't match this class.